src/components/pages/collectRecordFormPages/CollectRecordFormPageAlternative/CollectRecordFormPageAlternative.js

```diff
@@ -63,9 +65,8 @@ function loadObservationsFromCollectRecordIntoTableState({
     })
   }
 
-  if (collectRecordBeingEdited) {
-    const observationsFromApiTable =
-      collectRecordBeingEdited.data[getObservationsPropertyNames(collectRecordBeingEdited)[0]] ?? []
+  if (collectRecordBeingEdited && !isNewRecord) {
+    const observationsFromApiTable = collectRecordBeingEdited.data[observationsPropertyName] ?? []
```

Comment:
aah!!! the second obs table in bleacing wasnt showing data, because this was originally assuming the first obs table name in the array... which is for the first table, not the second